luna/examples/luna-theme-provider/src/ThemeShell/App.css

```
@@ -0,0 +1,55 @@
+@import "@lynx-js/luna-styles/index.css";
+
+.container {
+  width: 100%;
+  height: 100%;
+
+  display: flex;
```

Stylelint (17.14.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)